stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -40,6 +40,7 @@ struct MethodCompletion <: Completion
     func
     input_types::Type
     method::Method
+    orig_method::Union{Nothing,Method} # if `method` is a keyword method, keep the original method for sensible printing
```

Comment:
Without this, the completion would look like below and imho looks less friendly (while it's the internal truth):
```julia
julia> foo(1, kw1 = nothing, )
(::var"#foo##kw")(::Any, ::typeof(foo), a::Integer) in Main at none:1 (::var"#foo##kw")(::Any, ::typeof(foo), a) in Main at none:1
```

q: is there a way to retrieve the original `Method` object from its keyword-arg `Method` ? Then we don't need this additional field.